mathesar/api/db/viewsets/tables.py

```diff
@@ -54,6 +58,48 @@ def type_suggestions(self, request, pk=None):
         col_types = get_table_column_types(table)
         return Response(col_types)
 
+    @action(methods=['post'], detail=True)
+    def split_table(self, request, pk=None):
```

Comment:
I don't want to make those changes in this PR, we also have data explorer RPC API's that will be added soon, so I think it would be better to wait until then to have better clarity.

Reply:
Fair enough. Let's set up a meeting about that.